spec/models/site_spec.rb

```diff
@@ -602,14 +602,18 @@
     end
 
     describe "for threshold_for_referral" do
-      it "defaults to 50" do
-        allow(ENV).to receive(:fetch).with("threshold_for_referral", '50').and_return("50")
-        expect(defaults[:threshold_for_referral]).to eq(50)
+      it "defaults to 250" do
+        original_threshold = ENV["THRESHOLD_FOR_REFERRAL"]
+        ENV["THRESHOLD_FOR_REFERRAL"] = nil
+        expect(defaults[:threshold_for_referral]).to eq(250)
+        ENV["THRESHOLD_FOR_REFERRAL"] = original_threshold
```

Comment:
The current specs seem to be stubbing out the logic they're testing. But I don't know if this is overkill 😬 .